0001/README.md

```diff
@@ -159,15 +159,16 @@ Given that, server-initiated requests are NOT RECOMMENDED.
 
 ### Request ID
 
-CoAP message has a built-in Token, which is used to match responses to requests.
+CoAP message has a built-in Token used to match responses to requests.
 
 MQTT PUBLISH packet includes a Packet Identifier.
-However, Packet Identifiers are not stable; that means, when a PUBLISH packet is re-transmitted by MQTT broker or MQTT gateway, the Packet Identifier may be changed.
+However, Packet Identifiers are not stable.
+This means, when a PUBLISH packet is re-transmitted by MQTT broker or MQTT gateway, the Packet Identifier may be changed.
 Thus, MQTT Packet Identifiers are not suitable for matching responses to requests.
 
-KP over MQTT MUST NOT use Packet Identifiers to match responses to requests.
+KP MUST NOT use Packet Identifiers over MQTT to match responses to requests.
```

Comment:
The meaning changed a little: originally the phrase meant: "KP, when ran over MQTT...". On the right, it seems like MQTT refers to "Packet Identifiers".

Reply:
Fixed.